### DISCUSSION TITLE
How to use the cover reinforecers?

### DISCUSSION THREAD

**Discussion by GaryBoone** [thehighnibble/thehighnibble.github.io#111] [IMSAI8080esp replica]
As the cover reinforcers were added after the videos finished, there isn't a demo of how they are assembled. Do you glue them in place? What kind of glue? In what order? Some are differently shaped. Where do they go?

Thanks!

**Comment by donfroula** [comment]
Having just dealt with this, I can tell you what I did.

First, I laid all of the pieces out in pairs to match them correctly to each other and to the corresponding corner of the back cover plate. The small rectangular slots don't all match up, so you can use them to pair the correct pieces, making sure the slots are perpendicular to the direction of the screw as it passes through the metal housing.

I used "Gorilla Super Glue Gel" to cement the pieces in place. First, I glued the back piece in place. Then I positioned the nut inside the sandwich of the two reinforcers and the back cover, then I glued the second piece in place. 

I found it was impossible to insert the nut through the reinforcer slot after gluing the reinforcers in place, without enlarging the slot with a file. It was easier just to insert the nuts before gluing the second piece.

One of the sandwiches was holding the nut at an angle (see image). I found it difficult to align the nut square to the screw so the threads would catch. I needed to use a dental probe to position the nut so the screw would catch correctly. The other three screws caught with no issue.

The Gorilla Super Glue Gel sets in seconds, so be sure of your alignment. I held pressure on the piece for 45 seconds to assure good contact. I waited an hour for the glue to set before attempting to attach the metal frame. It ended up working quite well, although the reinforcers and frame mounting method are mechanically pretty weak. The reinforcers help a lot.

One thing, The smaller piece is designed to provide clearance for the micro USB cord connector. I glued it on the wrong side and had to find a connector style that cleared the larger reinforcing piece (see image showing incorrect placement). Be sure to position the small piece on the outside of the back cover.

Best,

Don

![IMG_0742B](https://github.com/thehighnibble/thehighnibble.github.io/assets/9483397/e5f4825d-ef0d-4f3e-b738-f2360e0e916c)

**Comment by nbreeden2** [comment]
I followed the same process as donfroula outlined. Make sure to not get any glue on the nut or in the threads; you want the nut floating in the slot, this makes getting the screws started easier.  On one IMSAI I had the same experience with inserting the nuts after the reinforcers were glued in place and had to use a small file to enlarge the slot. On the second one I built I glued them with the nut preinstalled. I glued the inside reinforcers, put a bit of tape over the slots on the inside, dropped in the nuts then glued on the outside reinforcers. This kept the nuts from falling through the slots. One two of the slots the nut was loose enough to still fall out after both reinforcers were glued in place. The other two held the nut captive.
I used an inexpensive super glue.  
The reinforcers are required as the original top was plastic and have enough flex it wouldn't break out the slots on the back plastic. The metal cover in the newer design is much nicer but is stiffer and requires the reinforcers (learned this the hard way).
The metal cover also adds some weight and make the unit feel more solid when standing up.
I stood the electronics assembly (the sandwich with the switches) on a couple of small blocks of wood to raise it from the work surface a bit. The metal wrap is slightly taller than the electronics assembly and is what the assembled unit sits on when completed. I used a small jewelers screwdriver in the top hole to help align the screw holes then inserted the screw on the bottom holes first. 
It's all a bit fiddley but the end result with the metal wrap looks much more like an original IMSAI and is very pleasing to the eye.
I highly recommend keeping the original shipping box as it's perfect to store/protect the unit should you even need too.

**Comment by GaryBoone** [comment]
Thanks, @donfroula and @nbreeden2, for the clarifications. Success!

Summarizing:
* The reinforcers sandwich the clear back cover.
* The reinforcers have to checked in advance and aligned carefully because their orientations and cutouts are slightly different depending on location.
* The smaller, oddly shaped reinforcer goes on the outside near the ESP32-PICO to make room for the USB cable.
* CA/Superglue works fine. Use gel to give some time for alignment.
* Recommended procedure: 1) Glue the reinforcers on the PCB side. 2) Put tape on the PCB side of those reinforcers over the nut holes. 3) Put the nuts in so they stick to the tape. 4) Glue the remaining, outside reinforcers, avoiding glue on the nuts.

An additional tip: The purpose of the reinforcers is to take load off of the cruciform cutout in the back panel. For that to work, the nut has to touch the reinforcer cutouts, ideally equally or less than it touches the cruciform slot edges. The other straight edge of the reinforcer should align with the cover edge. Therefore, what I did was 

1. Stand the IMSAI on edge. 
2. Check the nut alignment, ie, that the bolt can still screw into the nut.
3. Apply glue to the reinforcer. 
4. Put it in place so the the table edge ensures perfect alignment with the cover edge. 
5. Before the glue sets, push the reinforcer along the table right or left so that it pushes against a metal stop I put in the nut hole, aligning it. At first the stop was metal tweezers whose edge was flat and just fit in the whole when pushed through. For the outer reinforcers, the taped-in nut worked. You want to feel carefully to make sure the stopper/nut is pushed against the edge of the cruciform slot that the bolt will be pulling it to, and the reinforcer is pushing against the stopper/nut. Before the glue sets, check that the bolt can still screw into the nut.

I did this on a glass table so that if any glue leaked out, it wouldn't stick anything to the table and could easily be scratched off with blade.
